DigitalLearningSolutions.Web/Views/LearningPortal/Shared/_Dates.cshtml

```diff
@@ -84,8 +84,8 @@
     <hr class="nhsuk-u-margin-top-0 nhsuk-u-margin-bottom-2" aria-hidden="true" />
 
     <p aria-describedby="@Model.Id-name @Model.Id-evaluated-title" name="evaluated-date">
-      @(completedCourseViewModel.EvaluatedDate == null ?
-        Html.Raw("<span class='nhsuk-u-visually-hidden'>Not evaluated</span>-") :
-        Html.Raw($"{completedCourseViewModel.EvaluatedDate?.ToShortDateString()}"))
+    @(completedCourseViewModel.EvaluatedDate == null ?
+      Html.Raw("<span class='nhsuk-u-visually-hidden'>Not evaluated</span>-") :
+      Html.Raw($"{completedCourseViewModel.EvaluatedDate?.ToShortDateString()}"))

```

Comment:
I know this is existing code, but this seems wrong, I have no idea why it would be done like this.

Reply:
Just tried to refactor it and it seems like it was done to avoid having to write the <p> tags wrapping the content more than once. I could refactor it to be more like how we do it in other places, but it would make the code a bit longer, think it's worth doing?

Reply:
I think it's the equivalent of 
![image](https://user-images.githubusercontent.com/59561751/146526046-99e4fd6c-6797-4dd0-8508-2058cde01fae.png)

We shouldn't really be using Html.Raw unless we have to.

Reply:
Thanks, I didn't know about the "@:" to fix the error I was getting on the "-". Refactored it like this everywhere Html.Raw is used in this file